simd.hpp

```diff
@@ -97,12 +95,16 @@ using native = avx512;
 using native = avx;
 #elif defined(__SSE2__)
 using native = sse;
-#elif defined(__ARM_NEON)
+#elif defined(__ARM_NEON) && !defined(__ARM_FEATURE_SVE_BITS)
```

Comment:
NEON is defined for SVE supported archs too, but we don't want to use it as the native type.